webpack/assets/javascripts/react_app/components/notifications/drawer/notificationGroup.js

```diff
@@ -39,9 +40,17 @@ export default (
               onClickedLink={onClickedLink}
               key={notification.id}
               notification={notification}
-              onMarkAsRead={onMarkAsRead.bind(this, group)}
+              onMarkAsRead={onMarkAsRead.bind(this, group, [notification.id])}
```

Comment:
why is it a different api call? its either calling the existing api multiple times, or simply we change the server api to receive one or more id's ?